kotlin/bin/main/App.kt

```diff
@@ -0,0 +1,21 @@
+import com.algolia.search.client.ClientSearch
```

Comment:
you added this file @Aallam, is that expected?

Reply:
I renamed the file from `app.kt` to `App.kt`, to match the convention, and this line:
```kotlin
application {
     // Define the main class for the application.
     mainClass = 'AppKt'
 }
```

Reply:
I mean that there's a bin/App.kt and src/App.kt (both tracked) @Aallam

Reply:
Oh you are right, I removed the file under `bin/` and added an `.gitignore` file